SecurityPkg/Include/Protocol/MuTcg2Protocol.h

```diff
@@ -38,11 +38,19 @@ EFI_STATUS
   IN TPML_DIGEST_VALUES   *DigestList,
   IN EFI_TCG2_EVENT       *Event
   );
+
+typedef
+EFI_STATUS
+(EFIAPI *MU_TCG2_INIT_EVENT)(
+  IN OUT TCG_PCR_EVENT2_HDR  *Event,
+  IN UINT32                  EventSize
+  );
 // MU_CHANGE - END - Add a new protocol to support Log-only events.
 
 struct tdMU_TCG2_PROTOCOL {
-  UINT32               Version;
-  MU_TCG2_LOG_EVENT    Tcg2LogEvent;
+  UINT32                Version;

```

Comment:
Is there a reason not to update the protocol version? Consumers need something (either a version update or a protocol GUID update) to know if the API is present.

Reply:
Had to leave the comment here due to modified lines, but I'm referring to updating `MU_TCG2_PROTOCOL_VERSION` above in the file.